### PR TITLE
Hoang/stakeibc test refactor

### PR DESCRIPTION
**Pull request by ThanhNhann**
Closes: #XXX

## Author's Checklist

I have...

- [ ] Run and PASSED locally all GAIA integration tests
- [ ] If the change is contentful, I either:
    - [ ] Added a new unit test OR 
    - [ ] Added test cases to existing unit tests
- [ ] OR this change is a trivial rework / code cleanup without any test coverage

If skipped any of the tests above, explain.

## Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] manually tested (if applicable)
- [ ] confirmed the author wrote unit tests for new logic
- [ ] reviewed documentation exists and is accurate

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes? 
  - [ ] Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`?
  - [ ] This pull request updates existing proto field values (and require a backend and frontend migration)? 
  - [ ] Does this pull request change existing proto field names (and require a frontend migration)?
  How is the feature or change documented? 
      - [ ] not applicable
      - [ ] jira ticket `XXX` 
      - [ ] specification (`x/<module>/spec/`) 
      - [ ] README.md 
      - [ ] not documented

### CODE REVIEW

File: x/stakeibc/handler.go [hunk 1]
```diff
@@ -57,7 +56,7 @@ func NewHandler(k keeper.Keeper) sdk.Handler {
 			// this line is used by starport scaffolding # 1
 		default:
 			errMsg := fmt.Sprintf("unrecognized %s message type: %T", types.ModuleName, msg)
-			return nil, sdkerrors.Wrap(sdkerrors.ErrUnknownRequest, errMsg)
+			return nil, fmt.Errorf(`%s : unknown request`, errMsg)
 		}
```
Comment (ThanhNhann) [inline x/stakeibc/handler.go:60]: It should be: `return nil, fmt.Errorf("%s: unknown request", errMsg)`

File: x/stakeibc/keeper/icqcallbacks_validator_exchange_rate_test.go [hunk 2]
```diff
@@ -211,7 +211,7 @@ func (s *KeeperTestSuite) TestValidatorExchangeRateCallback_DelegationQueryFaile
 
 	err := stakeibckeeper.ValidatorExchangeRateCallback(s.App.StakeibcKeeper, s.Ctx(), tc.validArgs.callbackArgs, tc.validArgs.query)
 
-	expectedErrMsg := "ValidatorCallback: failed to query delegation, zone GAIA, err: Zone GAIA is missing a delegation address!: "
-	expectedErrMsg += "ICA acccount not found on host zone: failed to submit ICQ"
+	expectedErrMsg := "ValidatorCallback: failed to query delegation, zone GAIA, err: Zone GAIA is missing a delegation address!"
+	expectedErrMsg += "%!(EXTRA string=ICA acccount not found on host zone): failed to submit ICQ"
```
Comment (ThanhNhann) [inline x/stakeibc/keeper/icqcallbacks_validator_exchange_rate_test.go]: plz check more time of this

File: x/stakeibc/keeper/msg_server_claim_undelegated_tokens_test.go [hunk 3]
```diff
@@ -192,8 +192,8 @@ func (s *KeeperTestSuite) TestClaimUndelegatedTokens_NoEpochTracker() {
 	s.App.StakeibcKeeper.RemoveEpochTracker(s.Ctx(), epochtypes.STRIDE_EPOCH)
 
 	_, err := s.GetMsgServer().ClaimUndelegatedTokens(sdk.WrapSDKContext(s.Ctx()), &tc.validMsg)
-	expectedErr := "unable to build redemption transfer message: "
-	expectedErr += "Epoch tracker not found for epoch stride_epoch: epoch not found"
+	expectedErr := "Epoch tracker not found for epoch stride_epoch: epoch not found: "
+	expectedErr += "unable to build redemption transfer message"
 	s.Require().EqualError(err, expectedErr)
```
Comment (ThanhNhann) [inline x/stakeibc/keeper/msg_server_claim_undelegated_tokens_test.go]: Why we need upcase the first letter like this

File: x/stakeibc/keeper/msg_server_submit_tx.go [hunk 4]
```diff
@@ -35,18 +34,18 @@ func (k Keeper) DelegateOnHost(ctx sdk.Context, hostZone types.HostZone, amt sdk
...
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidChainID, "%s has no associated connection", portID)
+		return fmt.Errorf("%s has no associated connection: invalid chain-id", portID)
 	}
 
 	// Fetch the relevant ICA
 	delegationIca := hostZone.GetDelegationAccount()
 	if delegationIca == nil || delegationIca.GetAddress() == "" {
 		k.Logger(ctx).Error(fmt.Sprintf("Zone %s is missing a delegation address!", hostZone.ChainId))
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "Invalid delegation account")
+		return fmt.Errorf("Invalid delegation account: : invalid address")
```
Comment (ThanhNhann) [inline x/stakeibc/keeper/msg_server_submit_tx.go]: excess `:`

File: x/stakeibc/types/message_change_validator_weight_test.go [hunk 5]
```diff
@@ -20,20 +20,20 @@ func TestMsgChangeValidatorWeight_ValidateBasic(t *testing.T) {
 			msg: MsgChangeValidatorWeight{
 				Creator: "invalid_address",
 			},
-			err: sdkerrors.ErrInvalidAddress,
```
Comment (ThanhNhann) [inline x/stakeibc/types/message_change_validator_weight_test.go:23]: plz check this again

File: x/stakeibc/types/message_claim_undelegated_tokens.go [hunk 6]
```diff
@@ -44,19 +45,19 @@ func (msg *MsgClaimUndelegatedTokens) GetSignBytes() []byte {
...
 	if err != nil {
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "invalid creator address (%s)", err)
+		return fmt.Errorf("invalid creator address (%s): invalid address", err.Error())
 	}
 	// sender must be a valid stride address
 	_, err = utils.AccAddressFromBech32(msg.Sender, "stride")
 	if err != nil {
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "invalid sender address (%s)", err)
+		return fmt.Errorf("invalid sender address (%s): : invalid address", err.Error())
 	}
```
Comment (ThanhNhann) [inline x/stakeibc/types/message_claim_undelegated_tokens.go:54]: excess `:`

File: x/stakeibc/types/message_claim_undelegated_tokens_test.go [hunk 7]
```diff
@@ -33,7 +33,7 @@ func TestMsgClaimUndelegatedTokens_ValidateBasic(t *testing.T) {
 				HostZoneId: "GAIA",
 				Epoch:      uint64(1),
 			},
-			err: sdkerrors.ErrInvalidAddress,
+			err: fmt.Errorf("%s", &Error{errorCode: "invalid address"}),
```
Comment (ThanhNhann) [inline x/stakeibc/types/message_claim_undelegated_tokens_test.go]: Plz check this again

**Review by hoangdv2429 — APPROVED** [review]

**Pull request merged**